packages/amplify-graphql-index-transformer/src/__tests__/amplify-graphql-index-transformer.test.ts

```diff
@@ -43,14 +43,83 @@ test('throws if an invalid LSI is created', () => {
       id: ID! @primaryKey @index(name: "index1")
     }`;
 
+  const schemaEmptySortKeyFields = `
```

Comment:
I found that this test case was missed for when using the `sortKeyFields` directive on `primaryKey`, so I added that test as well.